cmd/make-virt-host/main.go

```diff
@@ -192,7 +199,12 @@ func main() {
 	}
 
 	var vbmcResult []VBMC
-	json.Unmarshal(vbmcOut, &vbmcResult)
+	_ = json.Unmarshal(vbmcOut, &vbmcResult)
+	if err != nil {
+		fmt.Fprintf(os.Stderr, "ERROR: Could not unmarshal details of vbmc: %s\n", err)
+		os.Exit(1)
```

Comment:
This looks like a mistake. We disregard the return but we still check `err`.

Reply:
Thanks, good catch, rebase mess